### ISSUE TITLE
requests for network

### ISSUE THREAD

**Issue by YukiZhang15** [david-gpu/srez#38]
Dear David Garcia,
     I'm so sorry to bother you . But I do not have other choices. Recently we try to use your github-srez repo for Face Identification of low resolution images. We don't know tensorflow very much, can not use GPU either because of the computer configuration. We want to implement it on mxnet or caffe. So may I trouble you to give us an exact simple picture of network structure based on the srez code, similar to input->conv -> max pooling -> fc1 -> softmax ?  If too busy, it can be painted directly on paper. 
    Thank you very much. So kind of you.
    Best wishes, looking forward to your reply.

**Comment by david-gpu** [comment]
The input data goes into [the generator](https://github.com/david-gpu/srez/blob/master/srez_model.py#L360). During training, the generator goes into [the discriminator](https://github.com/david-gpu/srez/blob/master/srez_model.py#L324). During inference the discriminator is not used at all; you just fetch the image produced by the generator.

There are lots of other implementations of image upscaling like this. For example, [arxiv.org/abs/1609.04802](https://arxiv.org/abs/1609.04802) describes a relatively similar network. More recently [arxiv.org/abs/1707.02921](https://arxiv.org/abs/1707.02921) came out, and they don't even use a GAN discriminator, so the network should be much easier to train.
